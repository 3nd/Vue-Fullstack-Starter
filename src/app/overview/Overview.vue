<template>
  <v-container>
    <v-layout column>
      <div>01. Hello World</div>
      <div>02. 101 Guide</div>
      <div>03. CRUD Operations</div>
      <div>04. Form Controls</div>
      <div>05. Data Table</div>
      <div>06. Internationalization</div>
      <div>07. Interactive Chart</div>
      <div>08. Authorization</div>
      <div>09. Payment</div>
      <div>10. Messages</div>
    </v-layout>
  </v-container>
</template>

<script>
export default {};
</script>

<style scoped>
</style>
