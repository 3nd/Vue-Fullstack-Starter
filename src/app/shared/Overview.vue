<template>
  <div>
    <v-jumbotron dark gradient="to bottom, #1E88E5, #42A5F5" height="auto">
      <v-container fill-height>
        <v-layout justify-center align-center wrap>
          <v-flex shrink>
            <img src="assets/images/logo.svg" alt="Logo" width="300" height="300" />
            <h3 class="display-3">Welcome to the site</h3>
            <span class="subheading">ABCD</span>
            <v-divider class="my-3"></v-divider>
            <div class="title mb-3">Check out our newest features!</div>
            <v-btn large color="white" to="/hello-world" class="primary--text mx-0">See more</v-btn>
          </v-flex>
        </v-layout>
      </v-container>
    </v-jumbotron>

    <v-container grid-list-xl class="mt-5 mb-5">
      <v-layout row justify-center>
        <v-flex xs4>
          <div class="elevation-1" style="height: 256px;">
            Standard Image
          </div>
        </v-flex>

        <v-flex xs4>
          <div>
            Standard Content
          </div>
        </v-flex>
      </v-layout>
    </v-container>

    <v-container grid-list-xl class="mt-5 mb-5">
      <v-layout row justify-center>
        <v-flex xs4>
          <div>
            Playground Content
          </div>
        </v-flex>

        <v-flex xs4>
          <div class="elevation-1" style="height: 256px;">
            Playground Image
          </div>
        </v-flex>
      </v-layout>
    </v-container>

    <v-container grid-list-xl class="mt-5 mb-5">
      <v-layout row justify-center>
        <v-btn large color="primary" to="/hello-world">See more</v-btn>
      </v-layout>
    </v-container>
  </div>
</template>

<script>
export default {
  metaInfo: {
    title: 'Vue by Example',
  },
};
</script>

<style scoped>
</style>
