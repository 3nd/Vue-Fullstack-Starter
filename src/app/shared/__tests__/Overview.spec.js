import { shallow } from '@vue/test-utils';

import Overview from '../Overview';

describe('Overview', () => {
  let [wrapper] = [];

  beforeEach(() => {
    wrapper = shallow(Overview);
  });

  it('should render light component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
