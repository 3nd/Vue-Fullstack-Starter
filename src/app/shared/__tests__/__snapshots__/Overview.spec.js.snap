// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview should render initial component 1`] = `
<v-layout fill-height="true" justify-center="true" align-center="true">
  <div class="text-xs-center">
    <div class="display-1 pa-3 primary--text">Vue by Example</div>
    <img src="&quot;logo.svg&quot;" alt="Logo" width="300" height="300">
  </div>
</v-layout>
`;
